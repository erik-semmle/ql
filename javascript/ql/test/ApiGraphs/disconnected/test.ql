import javascript

query API::Node sourceRead(string member) {
  exists(DataFlow::CallNode source | source.getCalleeName() = "source" |
    result = source.getAPINode().getMember(member)
  )
}
