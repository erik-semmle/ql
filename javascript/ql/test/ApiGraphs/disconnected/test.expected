| prop1 | index.js:4:17:4:25 | (no string representation) |
| prop2 | index.js:5:17:5:25 | (no string representation) |
